rero/rero-ils
# 1182 - improve perf with MEF

## Why are you opening this PR?

- Indexing becomes too slow in rero-ils.

## How to test?

To start:

```
git clone --depth=1 --single-branch --branch 1182-improve-perf-with-mef git@github.com:blankoworld/rero-ils
cd rero-ils
./scripts/bootstrap && docker-compose up -d && ./docker/wait-for-services.sh && ./scripts/setup
./scripts/server
```

To test:

  * check that UI:
    * contains authors
    * documents have authors
  * check /api with documents/authors (persons)
  * check anything else related to authors/persons!

## Code review check list
